cpp/test/Ice/objects/Test.ice

```diff
@@ -4,7 +4,8 @@
 
 #pragma once
 
-[["cpp:source-include:Forward.h"]]
+// The type of a C++ struct field must be complete.
+[["cpp:include:Forward.h"]]
```

Comment:
That's a new requirement. Previously, we used the complete type for a proxy used-as-a-type in the .cpp file (for encoding/decoding). Now we need it in the header _if_ we define fields of that proxy type.